lint/base_linter/node_linter.py

```diff
@@ -47,6 +47,22 @@ def smart_paths_upwards(start_dir):
     )
 
 
+def is_yarn_project(path, manifest):
+    # type: (str, Dict[str, Any]) -> bool
+    package_manager = manifest.get('packageManager')
+    if isinstance(package_manager, str):
```

Comment:
Is the intention to check for `None`s or can we have numbers and booleans here?

Reply:
`None`s.

Reply:
Okay.  Typically `if package_manager:` or `if pm is not None` but since this is user input your defensive approach makes totally sense.